stories/s1grdiw.eocloud.stories.js

```diff
@@ -41,17 +41,13 @@ export const getMapURL = () => {
   wrapperEl.innerHTML = '<h2>GetMapUrl (WMS)</h2>';
   wrapperEl.insertAdjacentElement('beforeend', img);
 
-  const layer = new S1GRDEOCloudLayer(
+  const layer = new S1GRDEOCloudLayer({
     instanceId,
     layerId,
-    null,
-    null,
-    null,
-    null,
-    AcquisitionMode.IW,
-    Polarization.DV,
-    Resolution.HIGH,
-  );
+    acquisitionMode: AcquisitionMode.IW,
+    polarization: Polarization.DV,
+    resolution: Resolution.HIGH,
```

Comment:
`resolution` isn't a parameter for `S1GRDEOCloudLayer` anymore

Reply:
Thanks, there was some mix-up (`resolution: OrbitDirection.ASCENDING`) with parameters too, so I changed all of them to `orbitDirection`. Fixed.

Reply:
Nice, resolving.